datacube/api/query.py

```diff
@@ -190,6 +192,15 @@ def _extract_time_from_ds(ds: Dataset) -> datetime.datetime:
 
 
 def query_group_by(group_by='time', **kwargs):
+    """
+    Group by function for loading datasets
+
+    :param str group_by: group_by name, supported str are:
```

Comment:
Note this function can also take a custom GroupBy object, which is simply returned untouched - worth documenting this.